src/instructlab/model/train.py

```diff
@@ -206,6 +206,18 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     required=True,  # default from config
     config_sections=ADDITIONAL_ARGUMENTS,
 )
+@click.option(
+    "--fsdp-cpu-offload-optimizer",
+    type=bool,
+    cls=clickext.ConfigOption,
+    help="Allow CPU offload for FSDP optimizer.",
```

Comment:
You don't need to setup `help` if you use the `clickext.ConfigOption` class it will be inherited from the Field description.